test/cpp/api/sequential.cpp

```diff
@@ -13,6 +13,10 @@ using namespace torch::test;
 
 struct SequentialTest : torch::test::SeedingFixture {};
 
+TEST_F(SequentialTest, CanContainThings) {
+  Sequential sequential(Linear(3, 4), Functional(torch::relu), BatchNorm(3));
```

Comment:
I think as part of the Python/C++ API parity project we want to encourage people to use `torch::nn::ReLU` and `F::relu` instead, to be more similar to the Python API. It would be awesome to change this to
```suggestion
  Sequential sequential(Linear(3, 4), ReLU(), BatchNorm(3));
```